src/Subscription/Activity.php

```diff
@@ -57,4 +61,27 @@ public function assertValid(Core\Activity $activity): void
             throw new SubscriptionIsForbidden('Your subscription is not valid for this activity');
         }
     }
+    /**
+     * @param int $id
+     */
+    public function persists(int $id): void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return \stdClass
+     */
+    public function jsonSerialize(): \stdClass
+    {
+        $object = new \stdClass();
+        $object->id = $this->id;
+        $object->type = get_class($this);
+        $object->startDay = $this->startDay;
+        $object->period = date_create('@0')->add($this->period)->getTimestamp();
+        $object->activity = $this->activity;
+        $object->visitor = $this->visitor;
+        $object->status = 'Available';
```

Comment:
What is the meaning of "status" for subscription to an activity?

Reply:
I have removed all unnecessary parametrs